src/brsTypes/components/RoSGNode.ts

```diff
@@ -1550,6 +1610,40 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
         },
     });
 
+    /* Checks whether the subtype of the subject node is a descendant of the subtype nodeType
+     * in the SceneGraph node class hierarchy.
+     *
+     *
```

Comment:
Looks like a few empty lines snuck in! 😃 Please remove these when you get a moment.